docs/integrations/vue.md

```diff
@@ -34,7 +34,7 @@ npm install gridjs-vue
 In `main.js` or wherever you specify your global components:
 
 ```js
-import Grid from 'gridjs-vue'
+import { GridGlobal } from 'gridjs-vue'
 
 Vue.use(Grid)
```

Comment:
Thanks for this PR, spent some time trying to figure out why is not working.
Consider change to Vue.use(GridGlobal) too.